Menus/TitleScreen.cpp

```diff
@@ -0,0 +1,608 @@
+					SetSectionDurationAndResetSwitch(38.0F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_SlideFadeInDuration = 0.5F;
+					m_SlideFadeOutDuration = 0.5F;
+				}
+				m_SlideshowSlideText = "Other civilizations were encountered...";
+				break;
+			case IntroSequence::ShowSlide6:
+				if (m_SectionSwitch) {
+					SetSectionDurationAndResetSwitch(44.1F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_SlideFadeInDuration = 0.5F;
+					m_SlideFadeOutDuration = 0.5F;
+				}
+				m_SlideshowSlideText = "...and peaceful intragalactic trade soon established.";
+				break;
+			case IntroSequence::ShowSlide7:
+				if (m_SectionSwitch) {
+					SetSectionDurationAndResetSwitch(51.5F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_SlideFadeInDuration = 0.5F;
+					m_SlideFadeOutDuration = 0.5F;
+				}
+				m_SlideshowSlideText = "Now, the growing civilizations create a huge demand for resources...";
+				break;
+			case IntroSequence::ShowSlide8:
+				if (m_SectionSwitch) {
+					SetSectionDurationAndResetSwitch(64.5F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_SlideFadeInDuration = 0.5F;
+					m_SlideFadeOutDuration = 0.5F;
+				}
+				if (m_SectionProgress < 0.3F) {
+					m_SlideshowSlideText = "...which can only be satisfied by the ever-expanding frontier.";
+				} else if (m_SectionProgress > 0.33F && m_SectionProgress < 0.64F) {
+					m_SlideshowSlideText = "Competition is brutal and anything goes in this galactic gold rush.";
+				} else if (m_SectionProgress > 0.67F) {
+					m_SlideshowSlideText = "You will now join it on a venture to an untapped planet...";
+				}
+				break;
+			case IntroSequence::SlideshowEnd:
+				if (m_SectionSwitch) {
+					SetSectionDurationAndResetSwitch(66.6F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_PreGameLogoText.SetPos(Vector(static_cast<float>(g_FrameMan.GetResX() / 2), static_cast<float>(g_FrameMan.GetResY() / 2)));
+					m_PreGameLogoTextGlow.SetPos(m_PreGameLogoText.GetPos());
+					m_GameLogo.SetPos(Vector(static_cast<float>(g_FrameMan.GetResX() / 2), static_cast<float>(g_FrameMan.GetResY() / 2) - 20));
+				}
+				break;
+			default:
+				break;
+		}
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void TitleScreen::UpdateIntroPreMainMenuSequence() {
+		switch (m_IntroSequenceState) {
+			case IntroSequence::GameLogoAppear:
+				if (m_SectionSwitch) {
+					SetSectionDurationAndResetSwitch(68.2F - static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS()));
+					m_IntroScrollStartTime = static_cast<float>(m_IntroSongTimer.GetElapsedRealTimeS());
+					m_IntroScrollDuration = 92.4F - m_IntroScrollStartTime; // 92.4s is the end of the planet scrolling
+					m_StationOrbitTimer.SetElapsedRealTimeS(40);
+					clear_to_color(g_FrameMan.GetOverlayBitmap32(), 0xFFFFFFFF); // White fade
```

Comment:
I don't think we need this comment. 0xFFFFFFFFFFFFFFFFFFFFFfFFFFFFFFFF is pretty obviously white (at least to anyone familiar with colour hexes)